packages/common/src/utils/string.ts

```diff
@@ -1,5 +1,5 @@
 const TAGS_REGEXP = /(<([^>]+)>)/gi;
-const TRAILING_UNDERSCORES_REGEXP = /^_+|_+$/g;
+const TRAILING_UNDERSCORES_REGEXP = /(^_+)|(_+$)/g;
```

Comment:
should fix a sonarcloud error that warned about this ambiguous pattern
https://sonarcloud.io/project/issues?issues=AX0phB3lhOt5ghKv_b89&open=AX0phB3lhOt5ghKv_b89&id=voiceflow_libs